### DISCUSSION TITLE
install this pi version to "regular" linux

### DISCUSSION THREAD

**Discussion by fredfred41** [mcguirepr89/BirdNET-Pi#632] [Q&A]
I search for this, but didn't find a solution.

Is there a way to install this Pi version to a "ragular linux, like latest ubuntu.

I know there is linux version of birdnet, but, if I understood right, it's not the same way to use it, it's command line only and made to analyze a whole bunch of audio recording. Am' I wrong? 

I would like to have the same way to use as BirdnetPi. Connecting and managing with a webpage.

Thanks

**Comment by blogdemoi** [comment]
You can install the Pi OS inside a virtual machine, I guess... Please, have a look at this article: https://pimylifeup.com/raspberry-pi-virtualbox/ (I never tried though).

**Comment by matthew73210** [comment]
Hey,
I've installed it on armbian so you aren't limited to rpi. If I remember correctly the installer checks if you are using an arm version. That might be a prob. If you can install an arm version of ubuntu you should be fine.
Cheers

**Comment by herve18** [comment]
Hello,
Thinking about installing BirdNet-Pi on a (very) old computer, running at 32 bits...but is this possible ?
Regards

**Reply by goobius** [reply]
Birdnet Pi requires 64bit ARM setup. I tried installing this on a virtual machine, PC, etc but could only get it to work with Armbian OS on an Raspberry Pi and Libre Computer board ( I posted about this board).
